lib/routes/dockerhub/repositories.ts

```diff
@@ -0,0 +1,41 @@
+import { Route, ViewType } from '@/types';
+import got from '@/utils/got';
+import { parseDate } from '@/utils/parse-date';
+import { Context } from 'hono';
+
+export const route: Route = {
+    name: 'DockerHub Repositories',
+    description: `List of repositories for an image owner`,
+    maintainers: ['CaoMeiYouRen'],
+    path: '/repositories/:owner',
+    categories: ['program-update'],
+    view: ViewType.Notifications,
+    example: '/dockerhub/repositories/library',
+    parameters: { owner: 'Image owner' },
+    handler,
+};
+
+async function handler(ctx: Context) {
+    const { owner, limits } = ctx.req.param();
```

Comment:
Please convert `owner` to lower case since that's the spec and user will still able to receive feed if they try something like `/dockerhub/repositories/DIYgod`